src/SprykerEco/Zed/Braintree/Communication/Plugin/Checkout/BraintreePostSavePlugin.php

```diff
@@ -18,6 +18,8 @@
 class BraintreePostSavePlugin extends BaseAbstractPlugin implements CheckoutPostSaveHookInterface
 {
     /**
+     * {@inheritDoc}
```

Comment:
Here should plugin specific specification.

Reply:
Updated.